app/Commands/System/SyncComponentsCommand.php

```
@@ -16,22 +16,23 @@ class SyncComponentsCommand extends Command
     protected $description = 'Sync component registry with installed packages (post-install hook)';
 
     public function handle(
-        ComponentManager $componentManager, 
+        ComponentManager $componentManager,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$componentManager'. (Unused Code Rules)

(UnusedFormalParameter)